### PR TITLE
[SPARK-48846][PYTHON][DOCS][FOLLOWUP]  Add a missing param doc in python api `partitioning` functions docs.

### PR DESCRIPTION
**Pull request by wayneguow**
### What changes were proposed in this pull request?

Add a missing param in func docs of `partitioning.py`.

### Why are the changes needed?

- Make python api docs better.

### Does this PR introduce _any_ user-facing change?

No.

### How was this patch tested?

Pass GA and docs check.

### Was this patch authored or co-authored using generative AI tooling?

No.

### CODE REVIEW

**Comment by wayneguow** [comment]
cc @HyukjinKwon

File: python/docs/.ruby-version [hunk 1]
```diff
@@ -0,0 +1 @@
+3.3.4
```
Comment (HyukjinKwon) [inline python/docs/.ruby-version:1]: why do we need this?

**Review by HyukjinKwon — APPROVED** [review]

File: python/docs/.ruby-version (hunk 1 above)
Comment (wayneguow) [reply python/docs/.ruby-version]: Sorry, made a mistake, I removed it.

**Comment by HyukjinKwon** [comment]
Merged to master.

**Pull request closed without merging**